README.md

```diff
@@ -96,7 +96,19 @@ FAR CRs are deleted by NHC after it sees the Node is healthy again.
 
 #### Example CR
 
-The FAR, `FenceAgentsRemediation`, CR is created by the admin and is used to trigger the fence-agent on a specific node. The CR includes an *agent* field for the fence-agent name, *sharedparameters* field with all the shared, not specific to a node, parameters, and *nodeparameters* field to specify the parameters for the fenced node.
+The FAR CR (CustomResource), created by the admin, is used to trigger the fence-agent on a specific node.
+
+The CR includes the following parameters:
+* `agent` - the fence-agent name
+* `sharedparameters` - all the shared, not specific to a node, parameters
+* `nodeparameters` - the parameters for the fenced node
+ 
+Optionally, the CR can include further parameters to specify how the operator should retry the fence-agent command in case of failure:
+
+* `.spec.retrycount` - the number of times to retry the fence-agent in case of failure. Default is 5.
+* `.spec.retryinterval` - the interval between retries in seconds. Default is "5s".
+* `.spec.timeout` - the timeout for the fence-agent in seconds. Default is "60s".
```

Comment:
I am not sure the `.spec.` prefix is needed. 

Can you also add these fields in the below YAML example (114-135)